scripts/check_crate_versions.sh

```
@@ -0,0 +1,54 @@
+
+# Check contracts
+for toml in contracts/*/*/Cargo.toml; do
+    if check_crate_version "$toml"; then
+        CONTRACT_CRATES+=($(dirname "$toml"))
+    fi
+done
+
+# Set outputs for GitHub Actions
+echo "macros_crate=$MACROS_CRATE" >> $GITHUB_OUTPUT
+echo "std_crate=$STD_CRATE" >> $GITHUB_OUTPUT
+echo "package_crates=${PACKAGE_CRATES[@]}" >> $GITHUB_OUTPUT
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)